src/common/structure/block.go

```diff
@@ -12,6 +12,8 @@ type Block struct {
 	RawBlock          interface{}
 	IsTaggable        bool
 	TagsAttributeName string
+	Lines             common.Lines
+	TagLines          common.Lines
```

Comment:
@tronxd Nice addition to the general block struct. Can you modify the Cloudformation block as well to use these members?